src/networks.json

```diff
@@ -1453,7 +1453,7 @@
     "chainId": 22040,
     "network": "testnet",
     "testnet": true,
-    "multicall": "0x18f7BFad831D8ED9E16C4cD50D6041dD453Ca1EF",
+    "multicall": "0xFCE5b5ca100e58E5635B15b79F0163be4CbD258b",
     "rpc": [
       "https://network-archive.ambrosus-test.io"
```

Comment:
@SvineruS This node is not an archive node, do you have any alternate node?

Reply:
:thinking: 
It's archive and working in https://snapshot-networks.on.fleek.co/
Also i can fetch balance on first block

Reply:
I should work for other blocks too, at least from the `start` block.

Reply:
it works for me. which block is not work for you?